server/index.ts

```diff
@@ -1,6 +1,6 @@
 import createApp from './app'
 import { services } from './services'
 
-const app = createApp(services)
+const app = createApp(services())
```

Comment:
we had a problem where importing services anywhere would cause the redis client to make a connection.
This would prevent tests from exiting properly so made this lazy so the connection is only called when the services function is actually called.

This allows us to import Service without creating connections